public/app/features/explore/Logs.tsx

```diff
@@ -460,5 +477,10 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
     radioButtons: css`
       margin: 0 ${theme.spacing.sm};
     `,
+    logsSection: css`
+      display: flex;
+      flexdirection: row;
+      height: 95vh;
```

Comment:
if we use `max-height` here and in https://github.com/grafana/grafana/pull/33259/files#diff-52f1275b53c883d4dcd6cfa6ab6c164a37f808e481f131abe880b4f09877b515R171 we could make it nicer in the situations you have only a few logs:

![Screenshot 2021-04-26 at 14 10 13](https://user-images.githubusercontent.com/1170767/116087785-2243ae00-a699-11eb-9a77-c7266642d34e.png)

instead of 
![Screenshot 2021-04-26 at 14 11 09](https://user-images.githubusercontent.com/1170767/116087926-430c0380-a699-11eb-87fa-8df51bcc2e6e.png)